simpleocr/files.py

```diff
@@ -1,9 +1,9 @@
 import os
 import cv2
-from tesseract_utils import read_boxfile, write_boxfile
+from simpleocr.tesseract_utils import read_boxfile, write_boxfile
 
 IMAGE_EXTENSIONS = ['.png', '.tif', '.jpg', '.jpeg']
-DATA_DIRECTORY = 'data/'
+DATA_DIRECTORY = os.path.join(os.path.dirname(os.path.realpath(__file__)), "data")
```

Comment:
A bit later than I had hoped, but I have taken a look and as far as I can tell (by testing it on Windows), the package works with `eggs` now by using `pkg_resources.resource_filename` to determine the path to an image file. Could you check on your own machine as well, just to make sure, @goncalopp ?